ide/editor.completion/src/org/netbeans/modules/editor/completion/PatchedHtmlRenderer.java

```diff
@@ -892,7 +892,7 @@ If NO (it's a closing tag)
                                 }
                             } else if (brutalWrap) {
                                 //wrap without checking word boundaries
-                                length = (new Double((w - widthPainted) / chWidth)).intValue();
+                                length = Double.valueOf((w - widthPainted) / chWidth).intValue();
```

Comment:
again, chWidth is a double ->
`(int) ((w - widthPainted) / chWidth))`

Reply:
Agreed. Changed.